src/main/java/domain/Score.java

```diff
@@ -0,0 +1,70 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import domain.board.Rank;
+import domain.piece.Pawn;
+import domain.piece.Piece;
+import domain.piece.team.Team;
+
+public class Score {
+	public static Map<Team, Double> calculateScore(List<Rank> ranks, Team... teams) {
+		Map<Team, Double> score = new HashMap<>();
+		Arrays.stream(teams)
```

Comment:
```suggestion
		return Arrays.stream(teams)
				.collect(Collectors.toMap(...));
```
toMap을 활용하면 좀더 깔끔하게 바뀔거에요

Reply:
고민하다가 우선 저렇게 적용했는데 toMap을 활용해서 수정해보도록 하겠습니다😀👍